src/Hintman/Config.hs

```diff
@@ -44,9 +44,15 @@ hintmanConfiguationCodec = HintmanConfig
 
 loadFileConfig :: FilePath -> IO HintmanConfig
 loadFileConfig path =
-    Toml.decodeFile hintmanConfiguationCodec path `catch` catchLoad
+    (Toml.decodeFile hintmanConfiguationCodec path `catch` catchLoad)
+      `catch` catchNoSuchFile
   where
     catchLoad :: LoadTomlException -> IO HintmanConfig
     catchLoad _ = do
         print $ "Hintman failed to load config from " <> path
         pure defaultHintmanConfig
+
+    catchNoSuchFile :: IOException -> IO HintmanConfig
+    catchNoSuchFile _ =  do
```

Comment:
To my taste: it would be nicer to use fancy `Exc` pattern-synonym from `relude` to handle multiple errors of different types:

* http://hackage.haskell.org/package/relude-0.4.0/docs/Relude-Exception.html#v:Exc

Reply:
Noted, I'll check relude